src/v2/view-builder/internals/BaseFooter.js

```diff
@@ -62,9 +63,16 @@ export default View.extend({
     }
 
     links.forEach(link => {
-      this.add(Link, {
-        options: link,
-      });
+      if (link.linkInfo) {
```

Comment:
Nice. this worked as i mentioned over slack. ❤️  that you created a new component, much cleaner.